### PR TITLE
amrfinderplus data_manager proper

### PR DESCRIPTION
**Pull request by pimarin**
amrfinderplus with no branch conflict !

### CODE REVIEW

File: data_managers/data_manager_build_amrfinderplus/test-data/amrfinderplus.loc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# this is a tab separated file describing the location of amrfinderplus database
```
Comment (bgruening) [inline data_managers/data_manager_build_amrfinderplus/test-data/amrfinderplus.loc:1]: shouldn't this file be named `.test` and the `/tmp/tmp3as7lw95/galaxy-dev`... can be removed isn't it?

File: data_managers/data_manager_build_amrfinderplus/tool-data/amrfinderplus.loc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# this is a tab separated file describing the location of amrfinderplus database
```
Comment (bgruening) [inline data_managers/data_manager_build_amrfinderplus/tool-data/amrfinderplus.loc:1]: this file should be named ".sample" I think and the example should be commented?

**Comment by pimarin** [comment]
Is it okay ?

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @pimarin

**Comment by pimarin** [comment]
Is it okay ?

**Comment by bgruening** [comment]
Yes!

**Comment by bgruening** [comment]
Thanks. Lets see if this works for real :)

**Pull request merged**